app/models/manageiq/providers/vmware/infra_manager/inventory/collector.rb

```diff
@@ -1,14 +1,14 @@
 class ManageIQ::Providers::Vmware::InfraManager::Inventory::Collector
+  include Connection
```

Comment:
I think these methods should exist in this class.

maybe don't make `@connection` an instance variable? (on the fence here)

Reply:
Agree, moving back